dipy/direction/bingham.py

```diff
@@ -0,0 +1,790 @@
+    weighted average of a given metric across the multiple ODF lobes
+    (weights are defined by the Bingham fiber density estimates).
+
+    Parameters
+    ----------
+    bmetric: ndarray(..., nl)
+        An arbitrary metric with values for nl ODF lobes.
+    bfd: ndarray(..., nl)
+        Bingham's fiber density estimates for the nl ODF lobes
+
+    Returns
+    -------
+    wmetric: ndarray(...)
+        Weight-averaged Bingham metric
+    """
+    return np.sum(bmetric * bfd, axis=-1) / np.sum(bfd, axis=-1)
+
+
+class BinghamMetrics:
+    """
+    Class for Bingham Metrics.
+    """
+
+    def __init__(self, model_params):
+        """Initialization of the Bingham Metrics Class.
+
+        Parameters
+        ----------
+        model_params : ndarray (..., nl, 12)
+            ndarray containing Bingham's model parameters fitted to ODFs
+            in the following order:
+            - Maximum value of the Bingham function (f0, index 0);
+            - concentration parameters k1 and k2 (indexes 1 and 2);
+            - elements of Bingham's main direction (indexes 3-5);
+            - elements of Bingham's dispersion major axis (indexes 6-8);
+            - elements of Bingham's dispersion minor axis (indexes 9-11).
+        """
+        self.model_params = model_params
+
+        self.peak_dirs = model_params[..., 3:6]
+
+    @auto_attr
+    def amplitude_lobe(self):
+        """Maximum Bingham Amplitude for each ODF lobe.
+        Measured in the unit 1/mm^3*rad."""
+        return self.model_params[..., 0]
+
+    @auto_attr
+    def kappa1_lobe(self):
+        """Concentration parameter k1 for each ODF lobe."""
+        return self.model_params[..., 1]
+
+    @auto_attr
+    def kappa2_lobe(self):
+        """Concentration parameter k2 for each ODF lobe."""
+        return self.model_params[..., 2]
+
+    @auto_attr
+    def kappa_total_lobe(self):
+        """Overall concentration parameters for an ODF peak.
+
+        The overall (combined) concentration parameters for each lobe is
+        defined by equation 19 in :footcite:p:`Tariq2016` as
+
+        .. math::
+            k_{total} = sqrt{(k_1 * k_2)}
+
+        References
+        ----------
+        .. footbibliography::
+        """
+        return np.sqrt(self.kappa1_lobe * self.kappa2_lobe)
+
+    @auto_attr
+    def odi1_lobe(self):
+        """Orientation Dispersion index 1 computed for each ODF lobe from
+        concentration parameter kappa1."""
+        return k2odi(self.kappa1_lobe)
+
+    @auto_attr
+    def odi2_lobe(self):
+        """Orientation Dispersion index 2 computed for each ODF lobe from
+        concentration parameter kappa2."""
+        return k2odi(self.kappa2_lobe)
+
+    @auto_attr
+    def odi_total_lobe(self):
+        """Overall Orientation Dispersion Index (ODI) computed for an
+        ODF lobe from the overall concentration parameter (k_total).
+        Defined by equation 20 in :footcite:p:`Tariq2016`.
```

Comment:
I believe the reason is `footcite` in a short description.  it can not be in a short description. so splitting short and long description like below should work. (I did not try, I can test this later tonight)

```python
"""Return Orientation Dispersion Index (ODI) for an ODF lobe.

The ODF lobe is from the overall concentration parameter (k_total).Defined by equation 20 in :footcite:p:`Tariq2016`.

References
----------
.. footbibliography::
"""
```

Reply:
Thanks, it appears to have worked.